### PR TITLE
Add round tests

### PR DESCRIPTION
**Pull request by JorjaF**
so here are the 10 tests that I did for round, i hope i got it right this time

### CODE REVIEW

File: spec/round_spec.rb [hunk 1]
```diff
@@ -6,12 +6,96 @@
...
+    round = Round.new(deck)
+    expect(round.guesses).to eq([])
+  end
+
+  it "has a current card" do
+    card_1 = Card.new("3","Hearts")
+    card_2 = Card.new("4", "Clubs")
+    deck = Deck.new([card_1, card_2])
+    round = Round.new(deck)
+    expect(round.current_card).to eq("3","Hearts")
```
Comment (dcoleman21) [inline spec/round_spec.rb]: Look at the interaction pattern for the return value of `round.current_card`. Is it expecting a string?

File: spec/round_spec.rb [hunk 2]
```diff
@@ -6,12 +6,96 @@
...
+  end
+
+  it "can record an incorrect guess" do 
+    card_1 = Card.new("3","Hearts")
+    card_2 = Card.new("4", "Clubs")
+    deck = Deck.new([card_1, card_2])
+    round = Round.new(deck)
+    new_guess = round.record_guess({value: "3", suit: "Hearts"})
+    new_guess2 = round.record_guess({value: "Jack", suit: "Diamonds"})
+    expect(round.guesses.count).to eq(2)
```
Comment (dcoleman21) [inline spec/round_spec.rb:70]: I think getting a count is good. I might write another `expect` statement as well to test what the interaction is showing. 

In the interaction pattern when it says
```ruby
pry(main)> round.guesses
#=> [#<Guess:0x00007f972a15c160 @card=#<Card:0x00007f972a227f18 @suit="Hearts", @value="3">, @response="3 of Hearts">]
```
What would this `expect` statement look like?

**Review by dcoleman21 — CHANGES_REQUESTED** [review]
This is looking good. I did leave you some comments. See if you can get those figured out.

**Comment by JorjaF** [comment]
I made some changes, i hope this is okay

File: spec/round_spec.rb (hunk 2 above)
Comment (dcoleman21) [reply spec/round_spec.rb:70]: What data type is the return value of round.guesses?

**Pull request merged**